geth/params/defaults.go

```diff
@@ -27,7 +27,7 @@ const (
 	ListenAddr = ":0"
 
 	// APIModules is a list of modules to expose via any type of RPC (HTTP, IPC, in-proc)
-	APIModules = "db,eth,net,web3,shh,personal,admin,debug"
+	APIModules = "db,eth,net,web3,shh,personal,debug"
```

Comment:
I would remove `db`, `personal` and `debug` as well. If one needs them, they can be turned on in the config.

Reply:
i think personal_sign/personal_ecrecover is commonly used in dapps. let's leave it.
i will remove two others